test/e2e/verifier.test.js

```diff
@@ -0,0 +1,228 @@
+import 'babel-polyfill';
+import { Status } from '../../config/default';
+import { VERIFICATION_STATUSES } from '../../src/index';
+import Certificate from '../../src/certificate';
+
+import EthereumMainV2Valid from '../fixtures/ethereum-main-valid-2.0';
+import EthereumMainInvalidMerkleRoot from '../fixtures/ethereum-merkle-root-unmatch-2.0';
+import EthereumMainRevoked from '../fixtures/ethereum-revoked-2.0';
+import EthereumRopstenV2Valid from '../fixtures/ethereum-ropsten-valid-2.0';
+import EthereumTampered from '../fixtures/ethereum-tampered-2.0';
+import MainnetInvalidMerkleReceipt from '../fixtures/mainnet-invalid-merkle-receipt-2.0';
+import MainnetMerkleRootUmmatch from '../fixtures/mainnet-merkle-root-unmatch-2.0';
+import MainnetV2Revoked from '../fixtures/mainnet-revoked-2.0';
+import MainnetV2Valid from '../fixtures/mainnet-valid-2.0';
+import MainnetV2AlphaValid from '../fixtures/mainnet-valid-2.0-alpha';
+import MocknetV2Valid from '../fixtures/mocknet-valid-2.0';
+import RegtestV2Valid from '../fixtures/regtest-valid-2.0';
+import Testnet404IssuerUrl from '../fixtures/testnet-404-issuer-url';
+import TestnetV1NoIssuerProfile from '../fixtures/testnet-no-issuer-profile-1.2';
+import TestnetRevokedV2 from '../fixtures/testnet-revoked-key-2.0';
+import TestnetTamperedHashes from '../fixtures/testnet-tampered-hashes-2.0';
+import TestnetV1Valid from '../fixtures/testnet-valid-1.2';
+import TestnetV2Valid from '../fixtures/testnet-valid-2.0';
+import TestnetV2ValidV1Issuer from '../fixtures/testnet-valid-v1-issuer-2.0';
```

Comment:
nice, we could even just add an index in the `fixtures` folder which we could import and reuse there without adding so many imports in this case

`import * as FIXTURES from '../fixtures'`

`FIXTURES.TestnetV2ValidV1Issuer` etc

Reply:
Yeah that crossed my mind after crossing the 10 import lines. Will do.